include/private/rdydmimpl.h

```diff
@@ -19,5 +19,6 @@ PETSC_INTERN PetscErrorCode CloneAndCreateCellCenteredDM(DM dm, const SectionFie
 PETSC_INTERN PetscErrorCode CreateDM(RDy rdy);
 PETSC_INTERN PetscErrorCode CreateAuxiliaryDM(RDy rdy);
 PETSC_INTERN PetscErrorCode CreateVectors(RDy rdy);
+PETSC_INTERN PetscErrorCode CreateSectionForSWE(RDy, PetscSection *);
```

Comment:
In the interest of confining SWE-specific code to `rdysweimpl.h` and `src/swe`, maybe we could call this `CreateSection` and have it dispatch to `CreateSWESection` in `rdysweimpl.h` (defined in `src/swe/physics_swe.c`, perhaps).

Reply:
I agree with the change you have suggested, but how about you make that change in the next operator refactor PR?